LEGO1/mxpalette.cpp

```diff
@@ -1,8 +1,47 @@
 #include "mxpalette.h"
 
+// OFFSET: LEGO1 0x100bee30
+MxPalette::MxPalette()
+{
+  this->m_overrideSkyColor = FALSE;
+  this->m_attached = NULL;
+  GetDefaultPalette(this->m_entries);
+  this->m_skyColor = this->m_entries[0x8d];
```

Comment:
Let's avoid hexadecimal when it's not needed.
```suggestion
  this->m_skyColor = this->m_entries[141];
```